understanding/timeouts.html

```diff
@@ -55,151 +53,107 @@
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 2.2.6:</span>Timeouts (Level AAA)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 2.2.6:</span>タイムアウト (レベル AAA)
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>Users are warned of the duration of any <a href="#dfn-user-inactivity">user inactivity</a> that could cause data loss, unless the data is preserved for more than 20 hours when
-                     the user does not take any actions.
+                  <p>データの損失を引き起こす恐れのある<a href="#dfn-user-inactivity">利用者の無操作</a>の残り時間が警告される。ただし、利用者が 20 時間以上何もしなくてもデータが保持される場合は、この限りではない。
+
                   </p>
                   <div class="note">
-                     <p class="note-title marker">Note</p>
-                     <p>Privacy regulations may require explicit user consent before user identification has
-                        been authenticated and before user data is preserved. In cases where the user is a
-                        minor, explicit consent may not be solicited in most jurisdictions, countries or regions.
-                        Consultation with privacy professionals and legal counsel is advised when considering
-                        data preservation as an approach to satisfy this success criterion.
+                     <p class="note-title marker">注記</p>
+                     <p>プライバシー関連の規制により、利用者を認証する前や利用者のデータが保存される前に、利用者の明確な同意が必要になる可能性がある。利用者が未成年の場合、ほとんどの司法管轄、国又は地域で、利用者からの明示的な同意を要請することができない。この達成基準に適合するためのアプローチとしてデータの保存を検討する場合は、プライバシーの専門家や弁護士に相談するのが望ましい。
                      </p>
                   </div>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   				
-                  <dt>Goal</dt>
-                  <dd>Users do not lose data due to unknown timeouts.</dd>
+                  <dt>目標</dt>
+                  <dd>利用者が不意なタイムアウトによってデータを失わないようにする。</dd>
                   				
-                  <dt>Author task</dt>
-                  <dd>Tell users how long their session can be inactive before they may lose information.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>利用者が情報を失う可能性がある前に、セッションが非アクティブになるまでの時間を伝える。</dd>
                   				
-                  <dt>Why it's important</dt>
-                  <dd>People with disabilities may need more time to complete actions.</dd>
+                  <dt>なぜそれが重要か</dt>
+                  <dd>障害のある人は、行為を完了するのに、より長い時間を要する可能性がある。</dd>
```

Comment:
人々にしようという話だったかと思いますので
```suggestion
                  <dd>障害のある人々は、行為を完了するのに、より長い時間を要する可能性がある。</dd>
```